tensorflow/python/ops/nn_impl.py

```diff
@@ -626,6 +626,18 @@ def l2_normalize_v2(x, axis=None, epsilon=1e-12, name=None):
 
   For `x` with more dimensions, independently normalizes each 1-D slice along
   dimension `axis`.
+  For example:
+  
+  ```python
+  import tensorflow as tf
```

Comment:
Import is not needed.

Reply:
Alright, I had to first cleanup my commit history then work on the changes now.